c7n/resources/ebs.py

```diff
@@ -50,27 +51,59 @@ class SnapshotAge(AgeFilter):
         days={'type': 'number'},
         op={'type': 'string', 'enum': OPERATORS.keys()})
     date_attribute = 'StartTime'
+    
+
+def _filter_ami_snapshots(self, snapshots):
+# gets a listing of all AMI snapshots and compares resources to the list
+    c = local_session(self.manager.session_factory).client('ec2')
+    ami_snaps = []
+    for i in c.describe_images(Owners=['self'])['Images']:
+        for dev in i.get('BlockDeviceMappings'):
+            if 'Ebs' in dev and 'SnapshotId' in dev['Ebs']:
+                ami_snaps.append(dev['Ebs']['SnapshotId'])
+    matches = []
+    if self.data.get('value', True):
+        for snap in snapshots:          
+            if snap['SnapshotId'] not in ami_snaps:
+                self.log.debug('ami-snapshot: %s' %snap['SnapshotId'])
```

Comment:
can we get rid of this, we typically run with debug verbose on, seems excessive given the other log about filter counts.